libs/server/Resp/KeyAdminCommands.cs

```diff
@@ -40,6 +40,46 @@ private bool NetworkRENAME<TGarnetApi>(ref TGarnetApi storageApi)
             return true;
         }
 
+        /// <summary>
+        /// TryRENAMENX
+        /// </summary>
+        private bool NetworkRENAMENX<TGarnetApi>(ref TGarnetApi storageApi)
+            where TGarnetApi : IGarnetApi
+        {
+            if (parseState.Count != 2)
+            {
+                return AbortWithWrongNumberOfArguments(nameof(RespCommand.RENAMENX));
+            }
+
+            var oldKeySlice = parseState.GetArgSliceByRef(0);
+            var newKeySlice = parseState.GetArgSliceByRef(1);
+            var status = storageApi.RENAMENX(oldKeySlice, newKeySlice);
```

Comment:
GarnetStatus is not meant to be interpreted this way, you should pass an output parameter to this method and write the appropriate output to it.

Reply:
Will do it

Reply:
Fixed